<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>571</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color:rgb(254, 224, 255);</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QFrame" name="frame_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>791</width>
        <height>531</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(254, 224, 255);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QFrame" name="frame_2">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>330</y>
         <width>351</width>
         <height>131</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(208, 209, 255);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>30</y>
          <width>251</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border-color: rgb(255, 255, 255);
font: 18pt &quot;Calibri&quot;;</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonE_2">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>80</y>
          <width>93</width>
          <height>29</height>
         </rect>
        </property>
        <property name="text">
         <string>Enter</string>
        </property>
       </widget>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>30</y>
         <width>691</width>
         <height>261</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
font: 300 18pt &quot;Calibri Light&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>380</y>
        <width>181</width>
        <height>141</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="pushButtonW">
       <property name="geometry">
        <rect>
         <x>32</x>
         <y>48</y>
         <width>51</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonS">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>94</y>
         <width>51</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonE">
       <property name="geometry">
        <rect>
         <x>128</x>
         <y>49</y>
         <width>51</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonN">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>2</y>
         <width>51</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>30</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">
background-color:rgb(254, 224, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="putButton">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>450</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(199, 203, 255)</string>
      </property>
      <property name="text">
       <string>Put</string>
      </property>
     </widget>
     <widget class="QPushButton" name="takeButton">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>450</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(199, 203, 255)</string>
      </property>
      <property name="text">
       <string>Take</string>
      </property>
     </widget>
     <widget class="QPushButton" name="helpButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(199, 203, 255)</string>
      </property>
      <property name="text">
       <string>Help</string>
      </property>
     </widget>
     <widget class="QPushButton" name="invButton">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>10</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(199, 203, 255)</string>
      </property>
      <property name="text">
       <string>Inventory</string>
      </property>
     </widget>
     <widget class="QPushButton" name="mapButton">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>10</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(199, 203, 255)</string>
      </property>
      <property name="text">
       <string>Map</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>481</width>
        <height>201</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="item2">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>80</y>
        <width>111</width>
        <height>111</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
background-color:rgb(254, 224, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="item1">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>80</y>
        <width>111</width>
        <height>111</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">
background-color:rgb(254, 224, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="item3">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>220</y>
        <width>111</width>
        <height>111</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
background-color:rgb(254, 224, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="item4">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>230</y>
        <width>111</width>
        <height>111</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
background-color:rgb(254, 224, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="useButton">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>450</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(199, 203, 255)</string>
      </property>
      <property name="text">
       <string>Use</string>
      </property>
     </widget>
     <widget class="QPushButton" name="invItem1">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>80</y>
        <width>111</width>
        <height>111</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="invItem2">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>80</y>
        <width>111</width>
        <height>111</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="invItem3">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>220</y>
        <width>111</width>
        <height>111</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="invItem4">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>230</y>
        <width>111</width>
        <height>111</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="exitInv">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>10</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(199, 203, 255)</string>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="openButton">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>450</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(199, 203, 255)</string>
      </property>
      <property name="text">
       <string>Open</string>
      </property>
     </widget>
     <widget class="QPushButton" name="smashButton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>450</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(199, 203, 255)</string>
      </property>
      <property name="text">
       <string>Smash</string>
      </property>
     </widget>
     <widget class="QPushButton" name="winB">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>310</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(199, 203, 255)</string>
      </property>
      <property name="text">
       <string>Win</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
